src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -319,6 +321,24 @@ def avg_awc(sjs_result):
 
 
 @shared_task(throws=Exception)
+def soiln(sjs_result):
+    """
+    Get `SoilN` from MMW-Geoprocessing endpoint
+
+    Originally a static value of 2000 at Class1.vb@1.3.0:9587
+    """
+    if 'error' in sjs_result:
```

Comment:
Wonder when it would make sense to rename `sjs_result` -> `geoprocessing_result`.

Reply:
Is it just this file that refers to it as such? I'd be happy to include a rename commit while working on the other two cards in this family.

Reply:
I'll add a lint commit to this PR that changes all of them. Good catch.